readme.md

```diff
@@ -8,20 +8,22 @@ This is a C/C++ code analysis plugin for Eclipse that shows bugs detected by the
 
 * Linux operating system
 * Recent [CodeChecker](https://github.com/Ericsson/codechecker)
-* [Eclipse](www.eclipse.org): the plugin is currently tested with Eclipse Neon, but any recent Eclipse version should work
+* The last supported Eclipse version is [Oxygen 1](https://www.eclipse.org/downloads/packages/release/oxygen/1), version 4.7.0.
```

Comment:
I would rephrase this a bit.

`Every eclipse version newer than [Oxygen 1](https://www.eclipse.org/downloads/packages/release/oxygen/1) v4.7.0 is supported.`